modules/ztp-acm-adding-images-to-mirror-registry.adoc

```diff
@@ -0,0 +1,45 @@
+// CNF-1500 ZTP - preparing to install ACM
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+[id="ztp-acm-adding-images-to-mirror-registry_{context}"]
+= Adding {op-system} ISO and RootFS images to the disconnected mirror host
+
+Use the disconnected mirror to host the {op-system-first} images you require to provision your DU bare-metal hosts.
+
+.Prerequisites
+
+* Deploy and configure a HTTPD server to host the {op-system} image resources on the network.
+
+.Procedure
+
+. Log in to the mirror host.
+. Download the required {op-system} installation ISO and RootFS images from link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/[mirror.openshift.com], and move them to a folder that is accessible over the network, for example, `/var/www/html`:
++
+[source,terminal]
+----
+$ sudo wget https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest-4.8/rhcos-4.8.0-fc.9-x86_64-live-rootfs.x86_64.img -O /var/www/html/rhcos-4.8.0-fc.9-x86_64-live-rootfs.x86_64.img
+----
++
+[source,terminal]
+----
+$ sudo wget https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest-4.8/rhcos-4.8.0-fc.9-x86_64-live.x86_64.iso -O /var/www/html/rhcos-4.8.0-fc.9-x86_64-live.x86_64.iso
+----
++
+.Verification steps
++
+* Verify that the images downloaded successfully and are being served on the disconnected mirror host:
++
+[source,terminal]
+----
+$ curl http://$(hostname)/rhcos-4.8.0-fc.9-x86_64-live-rootfs.x86_64.img
```

Comment:
Double-check that you're using the replaceable [correctly](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output):

> To mark up command syntax, use the code block and wrap any replaceable values in angle brackets (<>) with the required command parameter, using underscores (_) between words as necessary for legibility.

Reply:
Are you referring to `$(hostname)`? 

This $variable will resolve to the machine hostname. try it out locally `echo $(hostname)`.